packages/byods/src/token-manager/index.ts

```diff
@@ -22,14 +24,20 @@ export default class TokenManager {
    * @example
    * const tokenManager = new TokenManager('your-client-id', 'your-client-secret');
    */
-  constructor(clientId: string, clientSecret: string, baseUrl: string = PRODUCTION_BASE_URL) {
+  constructor(
+    clientId: string,
+    clientSecret: string,
+    baseUrl: string = PRODUCTION_BASE_URL,
+    tokenStorageAdapter: TokenStorageAdapter = new InMemoryTokenStorageAdapter()
```

Comment:
By default, we implement an in memory adapter. Users can implement the base class if they want to use another one.